projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.spec.ts

```diff
@@ -162,22 +169,22 @@ describe('BulkActionsDropdownComponent', () => {
         fixture.detectChanges();
       });
 
-      it('should run action on selection', () => {
+      it('should run action on selection', async () => {
         spyOn(component, 'runAction').and.callThrough();
         spyOn(extensionService, 'runActionById');
-        const option = getElement(mockItem.id);
-        option.click();
-        fixture.detectChanges();
+        loader = TestbedHarnessEnvironment.loader(fixture);
+        const selectHarness = await loader.getHarness(MatSelectHarness);
+        await selectHarness.open();
+        await selectHarness.clickOptions(await selectHarness.getOptions()[0]);
+        await fixture.whenStable();
 
-        expect(component.runAction).toHaveBeenCalledWith(mockItem);
-        expect(extensionService.runActionById).toHaveBeenCalledWith(mockItem.actions.click, {
+        expect(extensionService.runActionById).toHaveBeenCalledOnceWith(mockItem.actions.click, {
           focusedElementOnCloseSelector: '.adf-context-menu-source'
         });
       });
 
       it('should reset selection on bulkActionExecuted', () => {
-        const option = getElement(mockItem.id);
-        option.click();
+        getElement(mockItem.id).click();
```

Comment:
We can use the harness approach here as well, since its similar to the code above, that was replaced by the harness approach

Reply:
Changes to harnesses, thanks!